glific/floweditor
# Added feature for delay node in floweditor

## Summary 

Added a delay node. 

## Steps to test

1. In package.json change this "@nyaruka/flow-editor": "glific/floweditor#v1.11.8.2" to "@nyaruka/flow-editor": "mdshamoon/floweditor#feature/delay-node"
2. Run yarn install && yarn run floweditor
3. Restart frontend server
4. Check if there is a new node `wait for time` for selection in the dialog box.
5. Add the delay time in the modal and click ok. It will create a new node having the title "wait for time".


### Format of data that is sent to the backend

The data is saved in this format in the flowRevisions:

```
"nodes": [
    {
      "uuid": "3ea030e9-41c4-4c6c-8880-68bc2828d67b",
      "actions": [
        {
          "type": "wait_for_time",
          "uuid": "a77035ac-4111-42c5-b1a0-6701aef9d67e",
          "delay": "time in seconds"
        }
      ]
    },
]
